src/components/DevDummyLogin/index.js

```diff
@@ -156,9 +156,11 @@ const DevDummyLogin = ({resetClickTime = () => {}}) => {
 
           const userId = await JwtDecode(response.token).user_id;
           const anonymousUserId = await JwtDecode(response.anonymousToken).user_id;
-          setProfileId({
+          setAuth({
             anonProfileId: anonymousUserId,
-            signedProfileId: userId
+            signedProfileId: userId,
+            token: response.token,
+            anonymousToken: response.anonymousToken
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setAuth` function is now being called with more parameters than the previous `setProfileId` function. This includes `token` and `anonymousToken`, which were not previously included. Ensure that these new parameters are handled correctly in the `setAuth` function and that they do not introduce any security vulnerabilities, such as exposing sensitive token information.
```diff
- setProfileId({
-    anonProfileId: anonymousUserId,
-    signedProfileId: userId
- });
+ setAuth({
+    anonProfileId: anonymousUserId,
+    signedProfileId: userId,
+    token: response.token,
+    anonymousToken: response.anonymousToken
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->